app_demo.rb

```diff
@@ -0,0 +1,106 @@
+require './lib/order'
+require './lib/submitorder'
+require 'dotenv'
+Dotenv.load
+
+class Takeaway
+
+  attr_accessor :menu1, :order1, :name, :price, :menu_item1, :total, :to_mobile, :from_mobile, :message_body, :message, :account_sid, :auth_token, :submitorder1, :quantity
+
+  def initialize
+    @menu1 = Menu.new Dish
+  end
+
+  def add_dish
+    while true
+      puts 'Add dishes name and then price (press return twice to end order )'
+      @name = gets.chomp
+      break if name == ''
+      @price = gets.chomp
+      p menu1.add_dish name, price.to_f
+    end
+  end
+
+  def submit_order
+    puts "Please give mobile number in format +447914245451:"
+    @to_mobile = gets.chomp
+    @from_mobile  = ENV['FROMNO']
+    @message_body = 'Thank you! Your order was placed and will be delivered before 18:52'
```

Comment:
Metrics/LineLength: Line is too long. [89/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)